app/controllers/users_controller.rb

```diff
@@ -1,3 +1,25 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find params[:id]
+  end
```

Comment:
Xử lý trường hợp không tìm thấy nữa em nhé

Reply:
có redirect thì em nên cho ng ta cái mess để thân thiện ng dùng nhé
kiểu có chia tay cũng cần có lý do chớ =)) 

Hoặc chị thì chị sẽ đổi dùng find thành find_by nè

```suggestion
 def show
    @user = User.find_by id: params[:id]
    return if @user
    
    flash[:warning] = "User not found" # tự I18n nhé
    redirect_to home_path
  end
```